### ISSUE TITLE
AOS is not working when window scroll event is not fired

### ISSUE THREAD

**Issue by h3ckphy** [michalsnik/aos#723]
Hi.
I have a div with class name "page" inside my body tag.
My web page is organized so that scroll event is not fired on window object, but it is triggered on that .page element.
And AOS thinks that I'm not scrolling at all. I bet you have a line in your source files like window.addEventListener("scroll", doAOS) or document.addEventListener("scroll", doAOS).

Can you please add an option which specifies on which element's scroll event AOS should work?

**Comment by duanhaobo** [comment]
你的问题解决了吗？我也遇到同样的问题了。

**Comment by h3ckphy** [comment]
No, I'm not using AOS js anymore. Just make your html tag to be scrollable. Remove height: 100% from html and body.
